lib/open_project/costs/engine.rb

```diff
@@ -181,8 +181,8 @@ class Engine < ::Rails::Engine
                       embed_as: ::API::V3::CostEntries::WorkPackageCostsByTypeRepresenter,
                       show_if: -> (*) {
                         represented.costs_enabled? &&
-                          (current_user_allowed_to(:view_cost_entries) ||
-                          current_user_allowed_to(:view_own_cost_entries))
+                         (current_user_allowed_to(:view_cost_entries, context: represented.project) ||
```

Comment:
Line is too long. [102/80]<br>Use 2 (not 1) spaces for indenting an expression spanning multiple lines.